### PR TITLE
Implemented missing focus indicators in test app

### PR DESCRIPTION
**Pull request by athottungal**
This PR will add some focus indicators around some of the commonly used widgets in the test app that will enable a keyboard user to see the indicator while an element is in focus. Also to see best practices of keyboard navigation in the test app so that I can implement them in my own project that consumes CAM.

**What did I change**

1. Added a focus indicator in Button widget which has been used in Drag and Drop widget as Draggable
2. Added a focus indicator in the Link widget
3. Added focus indicator in the MenuItem and Spin Button widget

### CODE REVIEW

File: test-app/src/widgets/Button.js [hunk 1]
```diff
@@ -4,19 +4,10 @@ import AccessibilityModule from '@curriculumassociates/createjs-accessibility';
...
-      '_onMouseDown',
-      '_onMouseUp',
-      '_onClick'
-    );
+    _.bindAll(this, '_onFocus', '_onBlur', '_onClick');
     this.callBack = callBack;
     this.addEventListener('focus', this._onFocus);
     this.addEventListener('blur', this._onBlur);
-    this.addEventListener('mousedown', this._onMouseDown);
-    this.addEventListener('mouseup', this._onMouseUp);
```
Comment (drosenberg123) [inline test-app/src/widgets/Button.js:19]: - can you tell me why these listeners were removed?

Comment (athottungal) [reply test-app/src/widgets/Button.js:19]: I removed these handlers who were responsible for hiding the background including the borders which looked like something shouldn't happen when interacted
https://github.com/CurriculumAssociates/createjs-accessibility/pull/94/files#diff-9b7586c142515f4edc22cfd6efdfeead4fc4240f14f1a06d08b36e31b99eebdcL94-L101

**Review by mmoss — CHANGES_REQUESTED** [review]

File: test-app/src/widgets/Button.js [hunk 2]
```diff
@@ -109,8 +84,10 @@ export default class Button extends createjs.Container {
     this.background.name = 'background';
     this.background.graphics
       .beginStroke('black')
-      .beginFill('#fff')
-      .drawRect(0, 0, this.width, this.height);
+      .beginFill('yellow')
```
[the remark this answers is not in the document]
Comment (athottungal) [reply test-app/src/widgets/Button.js:87]: I added a background just to differentiate between the focused and non-focused buttons. Here is the screen grab for both scenarios. 

**With some background:**
Screenshot 2022-07-08 at 1 43 35 PM

**Without background**
Screenshot 2022-07-08 at 1 47 44 PM

We can add some other color as a replacement for yellow, @mmoss Please let me know your thoughts

[the remark this answers is not in the document]
Comment (mmoss) [reply test-app/src/widgets/Button.js:87]: For context this is the WCAG criteria that we're targeting: https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html

**Review by mmoss — APPROVED** [review]
Looks like the focus indicator is aligned with how we're using it in other places now

Screen Shot 2022-07-13 at 10 41 05 AM

**Review by drosenberg123 — APPROVED** [review]
LGTM 👍 

thanks @athottungal!

**Pull request merged**